src/detect.js

```diff
@@ -40,21 +40,25 @@ function getUsedExportMap(includedFileMap, compilation) {
   compilation.chunks.forEach(function(chunk) {
     for (const module of chunk.modulesIterable) {
       if (!module.resource) continue;
-
-      const providedExports = module.providedExports || module.buildMeta.providedExports;
+      const providedExports = compilation.chunkGraph.moduleGraph.getProvidedExports(module);
```

Comment:
Yes. I think this would have to be taken care of by either doing some detection of webpack version (I can try to figure out a way) or publishing a new major version of the package to signal a breaking change. Let me know what you think should be done here.

Reply:
I think it is better if we can deal with it in this pr :D

Reply:
Is there any plan to go on with this feature?

Reply:
Sorry for the delay. I'm planning to if up this MR this weekend.

Reply:
Updated and tested on webpack 3 and webpack 5, both working fine. @MQuy let me know if you have any other concerns.

Reply:
thanks @paradite 👍🏻